app/scripts/app.js

```diff
@@ -29,6 +29,8 @@ angular
     localStorageServiceProvider.setPrefix('testReporter');
   }])
   .config(['$httpProvider', function ($httpProvider) {
+    $httpProvider.defaults.useXDomain = true;
```

Comment:
We don't need this anymore as we're proxying requests through express/connect

Reply:
Removed these lines